src/Tests/dotnet-publish.Tests/GivenDotnetPublishPublishesProjects.cs

```diff
@@ -40,7 +43,7 @@ public void ItPublishesARunnablePortableApp()
                 .Execute("--framework", ToolsetInfo.CurrentTargetFramework)
                 .Should().Pass();
 
-            var configuration = Environment.GetEnvironmentVariable("CONFIGURATION") ?? "Debug";
+            var configuration = Environment.GetEnvironmentVariable("CONFIGURATION") ?? _expectedConfigurationDefault;
```

Comment:
Maybe rename to `defaultConfiguration`? Calling it `_expected` makes it seem like it's intended to be a test result, but it's used as a default for when the environment variable is not set.

Reply:
Good call, this name used to be more relevant but that's better now.